wolfssl/io.h

```diff
@@ -234,6 +234,9 @@
 #elif defined(FREERTOS_TCP)
     #define RECV_FUNCTION(a,b,c,d)  FreeRTOS_recv((Socket_t)(a),(void*)(b), (size_t)(c), (BaseType_t)(d))
     #define SEND_FUNCTION(a,b,c,d)  FreeRTOS_send((Socket_t)(a),(void*)(b), (size_t)(c), (BaseType_t)(d))
+#elif defined(WOLFSSL_VXWORKS)
```

Comment:
If this fall into #else, 
#define HAVE_SOCKADDR part causes duplicated define of
typedef struct sockaddr         SOCKADDR;

They are defined in their header.